src/hotspot/cpu/aarch64/continuationHelper_aarch64.inline.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Note, you changed the initial year. Should be:
```diff
- * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2022, 2023, Oracle and/or its affiliates. All rights reserved. 
```

Reply:
My sed script which updated all copyright headers to 2023 didn't take into account files with only one year in the copyright line. My bad.

Reply:
#
foreach f (`cat file.list`)
  echo $f
  sed -e "s/Copyright (c) \([12][0-9][0-9][0-9]\), [0-9][0-9][0-9][0-9], Oracle/Copyright (c) \1, 2023, Oracle/"  \
      -e "s/Copyright (c) \([12][0-9][013456789][012456789]\), Oracle/Copyright (c) \1, 2023, Oracle/" <$f >$f.new
  diff $f $f.new
  mv $f.new $f
end